src/utils/user_utils.js

```diff
@@ -21,7 +21,7 @@ export function displayUsername(
     user: UserProfile,
     teammateNameDisplay: Preferences.DISPLAY_PREFER_NICKNAME | Preferences.DISPLAY_PREFER_FULL_NAME
 ): string {
-    let name = '';
+    let name = General.UNKNOWN_USERNAME;
```

Comment:
Instead of adding default username here i could do it for typing component alone by changning below selector if you guys prefer i am 1/5 on this. https://github.com/mattermost/mattermost-redux/blob/b1e1ec91c176d895044b3923b7ce4d6de0dc03af/src/selectors/entities/typing.js#L22

There are more than few places this util function is used so, i am not sure which is more favourable a empty srting or default string.

Reply:
I think this cover all cases no just for the use typing